aiida_quantumespresso/workflows/pdos.py

```diff
@@ -219,14 +222,15 @@ def define(cls, spec):
             help='Terminate workchain steps before submitting calculations (test purposes only).'
         )
         spec.input(
-            'align_to_fermi',
-            valid_type=orm.Bool,
+            'fermi_energy_range',
```

Comment:
I might be saying something stupid because I wasn't part of the discussions, but does the name `fermi_energy_range` really make sense? The values define the energy window in which the DOS is computed. Does something like just `energy_range` or `energy_window` not make more sense?

Reply:
I see your point. If a energy window is specified by this parameter, the DOS/PDOS is computed in this range with respect to the fermi energy. For example, the range 0 - 30 eV is specified and the fermi energy is 7 eV, then the DOS would be computed in the range 7 - 37 eV. Therefore, I think it would be useful to indicate this also by the name of the input parameter (but maybe the description in the help keyword would be already enough).

Reply:
To me it reads a bit weird having it in the option, so I would vote to put it simply in the description that it is relative to the fermi level. I will let @mbercx have the final vote.

Reply:
Sorry for being later to the party! Although I agree the name isn't perfect, I still think it's better to highlight that it's an energy range around the Fermi level by adding it to the name. Maybe `energy_range_fermi` would give less of an impression that there is some range of Fermi levels considered?

Reply:
Personally, I like the last one but I leave this up to you

Reply:
I also vote for the latter :+1:  A few more characters is almost always preferable if it adds clarity. So @t-reents please feel free to go with that one, update the PR and we can merge. Thanks!

Reply:
The changes are now included